package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonImporter.cs

```diff
@@ -224,39 +233,42 @@ public static class SkeletonImporter {
       }
       smr.bones = bones;
 
-      int[] indices = meshBinding.jointIndices.value;
-      float[] weights = meshBinding.jointWeights.value;
+      int[] indices = IntrinsicTypeConverter.FromVtArray((VtIntArray)jointIndices.GetAttr().Get());
+      float[] weights = IntrinsicTypeConverter.FromVtArray((VtFloatArray)jointWeights.GetAttr().Get());
```

Comment:
`GetAttr()` can return an invalid UsdAttribute, which might segfault (I think)
`Get()` can also fail and return an empty VtValue, which I think will then fail to cast

Again are these time varying? If so and we do not yet support that, it might be good to add a comment here.

Reply:
"Bon weights" -> "Bone weights"

Btw, if the `GetAttr()` call returns an invalid attribute, I think it may segfault. So you want something like this:

```
UsdAttribute jointIndicesAttr = jointIndices.GetAttr()
if (jointIndicesAttr.IsValid()) {
   VtValue indicesValue = jointIndicesAttr.Get();
   if (!indicesValue.IsEmpty()) {
     indices = IntrinsicTypeConverter.FromVtArray((VtIntArray)indicesValue)
   }
}
```

Reply:
But I can't see any case where it would return an invalid attribute. Do you have one in mind?